bit/wallet.py

```diff
@@ -605,7 +601,7 @@ def get_unspents(self):
             NetworkAPI.get_unspent_testnet(self.address)
         ))
         self.unspents += list(map(
-            lambda u: u.set_type('np2wpkh'),
+            lambda u: u.set_type('np2wkh'),
```

Comment:
Can you explain these changes?

Reply:
With Segwit a "type" on the Unspent objects was introduced. The type is mostly used to more accurately calculate the transaction size beforehand and thereby making a better fee estimation. There are many differences in naming the numerous script types between different implementations. NP2WKH denotes the nested Segwit P2PKH (also called P2SH-P2PKH, or NP2WPKH). I have settled with "np2wkh" in Bit, however made a typo here.

Reply:
Gotcha, sounds good.